bogenliga/src/app/modules/wettkampf/components/wettkampf/wettkampergebnis/WettkampfErgebnis.Service.ts

```diff
@@ -21,7 +21,7 @@ export class WettkampfErgebnisService {
   public veranstaltung: VeranstaltungDO;
 
   // Output
-  public wettkampErgebnisse: WettkampfErgebnis[] = [];
+  public wettkampErgebnisse: WettkampfErgebnis[];
```

Comment:
Klasse bitte umziehen in src/app/modules/wettkampf/services/wettkampf-ergebnis.service.ts (gibts schon!)

Reply:
Sollte jetzt nur noch eine Datei sein im Serviceordner. Zusätzlich noch Rechtschreib - und Formatkorrekturen